Jenkinsfile

```diff
@@ -54,5 +55,24 @@ pipeline {
                 archiveArtifacts artifacts: "$APK_PATH/*.apk", onlyIfSuccessful: true
             }
         }
+        stage('checkout test project repo') {
+            steps {
+                sh "mkdir -p $TEST_PROJECT_REPO_NAME"
+                dir("$TEST_PROJECT_REPO_NAME"){
+                    checkout([$class: 'GitSCM', branches: [[name: '*/master']], doGenerateSubmoduleConfigurations: false, extensions: [], submoduleCfg: [], userRemoteConfigs: [[url: 'https://github.com/edx/edx-app-test.git']]])
```

Comment:
For readability, can you add some indentation here?

Reply:
added, thanks